### DISCUSSION TITLE
Guidance on Geometric Parameters for Scaling LFP Battery Design

### DISCUSSION THREAD

**Discussion by jitendraec** [pybamm-team/PyBaMM#4436] [General]
I am working on scaling up an LFP battery cell design, using the Prada2013 parameters as a reference. The original cell has a capacity of 2.3 Ah, and I am aiming to develop a 72 Ah cell. I applied a scale-up factor of 72/2.3 to base my calculations.

My question is: as I scale up the nominal capacity, will the geometric parameters like electrode and separator thickness change? Or will only the length and width of the electrode need adjustment while maintaining same the thickness?

Any clarification on how these parameters should evolve during scaling would be greatly appreciated.

Thank you!

**Comment by rtimms** [comment]
Duplicate of #4446
